packages/components/inputs/date-time-input/src/date-time-input.tsx

```diff
@@ -186,8 +191,14 @@ class DateTimeInput extends Component<
       this.props.value === ''
         ? null
         : getDateInMonth(this.props.value, this.props.timeZone) - 1,
-    timeString: '',
+    timeString: this.props.defaultDaySelectionTime
+      ? formatDefaultTime(
+          this.props.defaultDaySelectionTime,
+          this.props.intl.locale
+        )
+      : '',
```

Comment:
Just a side note - this computation is executed only once, before the first render. Any changes of the `defaultDaySelectionTime` prop won't make difference. It's unlikely that this prop will change and this computation should be done again, so I'm not sure we should consider that a bug. But in that case I would definitely delete the `defaultDaySelectionTime` knob from both stories, since it won't ever work and gives impression that the component is buggy.

Reply:
I also think that's not a use case we need to worry about and removing the knob from the story is a good idea.

Reply:
amazing, i am removing the knob (from date-time-field as well)